certified-connectors/MAQTextAnalytics/apiDefinition.swagger.json

```diff
@@ -0,0 +1,250 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Text Analytics",
+    "description": "Discover meaningful insights in your text data. Perform Sentiment Analysis, extract Key Phrases and scrub PII Data from your text data.",
+    "contact": {
+      "name": "MAQ Software Sales",
+      "url": "https://maqsoftware.com/contact",
+      "email": "Sales@MAQSoftware.com"
+    }
+  },
+  "host": "maqtextnalyticssdk.azure-api.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/text/SentimentClassifier": {
+      "post": {
+        "summary": "Sentiment Classifier",
+        "description": "Perform Sentiment Analysis of your text data",
+        "operationId": "SentimentClassifier",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "id"
+                      },
+                      "text": {
+                        "type": "string",
+                        "description": "text"
+                      }
+                    }
+                  },
+                  "description": "data"
+                }
+              },
+              "default": {
+                "data": [
+                  {
+                    "id": 5,
+                    "text": "store wont launch but its coo when it worked"
+                  },
+                  {
+                    "id": 1,
+                    "text": "the best product"
+                  },
+                  {
+                    "id": 2,
+                    "text": "Very easy to understand and use the design lab!"
+                  },
+                  {
+                    "id": 3,
+                    "text": "Due to the COVID 19 pandemic or Trump Plague I would not recommend a visit to any store at this time. My order was place on line for an item I have used before and new exactly was I was looking for. Otherwise, I would recommend a visit to the store after the Trump Plague passes in order to see whatever product is too be purchased."
```

Comment:
Can you please make sure that the text is free from grammar errors? And I think when you say, "I have used before and new exactly", maybe you mean "**knew** exactly".